src/components/forecast.tsx

```
@@ -0,0 +1,75 @@
+import { CloudSunIcon, CompassIcon } from "lucide-react"
+
+
+const Forecast = () => {
+    return (
+        <div className='mt-10'>
+            <h1 className='heading-1'>
+                3 Day Forecast
+            </h1>
+            <div className='flex items-center gap-5'>
+                {Array.from({ length: 3 }).fill(0).map(() => (
+                    <ForecastCard />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)